apps/web/pages/teams/index.tsx

```diff
@@ -41,6 +42,20 @@ function Teams() {
 export const getServerSideProps = async (context: GetServerSidePropsContext) => {
   const ssr = await ssrInit(context);
   await ssr.viewer.me.prefetch();
+  const session = await getServerSession({ req: context.req, res: context.res });
+  const token = Array.isArray(context.query?.token) ? context.query.token[0] : context.query?.token;
+
+  const callbackUrl = token ? `/teams?token=${encodeURIComponent(token)}` : null;
+
+  if (!session) {
+    return {
+      redirect: {
+        destination: callbackUrl ? `/auth/login?callbackUrl=${callbackUrl}` : "/auth/login",
+        permanent: false,
+      },
+      props: {},
+    };
+  }
```

Comment:
this was removed when this PR https://github.com/calcom/cal.com/pull/11009 was reverted

Reply:
How is an invite to a team of org supposed to work? I copied the invite link and opened in a new incognito window to join by that link. But there was no session so it just redirected me to login page.